javalib/src/main/scala/java/time/temporal/package.scala

```diff
@@ -0,0 +1,11 @@
+package java.time
+
+package object temporal {
+  implicit class RichString(val s: String) extends AnyVal {
```

Comment:
This should be `private[time]`. Also, is it really worth having an implicit class here? I would recommend having a `private[time] def toScreamingSnakeCase(s: String): String` instead.